examples/tools/system/package_manager/ci_test_example.py

```diff
@@ -0,0 +1,40 @@
+from test.examples_tools import run
+import re
+import os
+import sys
+
+
+print("- Packaging system ncurses library using Conan -")
+
+if sys.platform != "linux":
+    print("INFO: Skipping test, only for Linux due to system requirements.")
+    sys.exit(0)
+
+confs = ["tools.system.package_manager:mode=install",
+         "tools.system.package_manager:sudo=true",
+         "tools.build:verbosity=verbose",
+         "tools.compilation:verbosity=verbose"]
+
+out = run("conan create . {}".format(" ".join(["-c " + conf for conf in confs])))
+
+assert "ncurses/system: System requirements"
+assert "package(): WARN: No files in this package" in out
+
+print("- Consuming Conan package ncurses/system -")
+
+out = run("conan create consumer/ --version=0.1.0 {}".format(" ".join(["-c " + conf for conf in confs])))
+
+assert "Conan: Target declared 'ncurses::ncurses'" in out
+assert "package(): Packaged 1 file: ncurses_version" in out
+
+match = re.search(r"Full package reference:\s*(.+)", out)
+assert match
+full_package_ref = match.group(1)
+
+package_folder = run(f"conan cache path {full_package_ref}").strip()
+assert os.path.isdir(package_folder)
+ncurses_version = os.path.join(package_folder, "bin", "ncurses_version")
+assert os.path.isfile(ncurses_version)
+
+out = run(ncurses_version)
```

Comment:
I think this is a bit more complicated than it should:
- Use a simpler consumer recipe, no name, no version, no ``package_info()``
- Have a ``build()`` method that does the build and also ``self.run(..., env="conanrun")`` the executable
- call ``conan build .``

Reply:
Done, only running `conan build` now in order to build the consumer: [fef2c93](https://github.com/conan-io/examples2/pull/177/commits/fef2c93cd2987861166ddcb7d9c1fca8c3e39831)